src/js/modules/supervisors/HandleError.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  DisableCoprosReply,
+  EnableCoprosReply,
+} from "../../modules/domain/generatedRpc/enableDisableCoprocClasses";
+import { Coprocessor } from "../../modules/public/Coprocessor";
+
+export enum ResponseEnableCode {
+  success,
+  internalError,
+  invalidIngestionPolicy,
+  scriptIdAlreadyExist,
+  topicDoesNotExist,
+  invalidTopic,
+  materializedTopic,
+}
+
+export enum ResponseDisableCode {
+  success,
+  internalError,
+  scriptDoesNotExist,
+}
+
+export const validateDisableCodeResponse = (
+  responses: DisableCoprosReply,
+  coprocessor: Coprocessor[]
+): Error[] => {
+  return responses.inputs.reduceRight<Error[]>((errors, code, index) => {
+    switch (code) {
+      case ResponseDisableCode.internalError: {
+        errors.push(
+          new Error(
+            `internal error: wasm id function: ` + coprocessor[index].globalId
```

Comment:
Since you're using template strings, shouldn't this be
```js
            `internal error: wasm id function: ${coprocessor[index].globalId}`?
```

Reply:
Please change the error message to
`internal error: wasm function ID: ${coprocessor[index].globalId}`

Reply:
Done